.env.example

```diff
@@ -0,0 +1,10 @@
+# DB_TYPE can be "sqlite", "mysql" or "postgres"
+DB_TYPE="postgres"
+# Only for sqlite:
+DB_STORAGE="db.sqlite"
+# Only for mysql and postgres:
+DB_HOST="localhost"
+DB_NAME="<%- dbname %>"
```

Comment:
Please Set a default name instead of keeping this template syntax

Reply:
Got it